@stellar/typescript-wallet-sdk/CHANGELOG.MD

```diff
@@ -0,0 +1,81 @@
+# Release notes - Typescript Wallet SDK - 1.4.0
```

Comment:
Not a big fan of a changelog in MD files considering we have it on Github, but we can keep it (still suggest to duplicate it into GH releases)

Reply:
Noted 🤔 we can discuss further if needed and if it's annoying duplicating it

Tbh the md files is where I go to first to see changes. Not sure what other people do though